services/order.py

```diff
@@ -0,0 +1,35 @@
+from django.db import transaction
+from django.db.models.query import QuerySet
+from services.movie_session import get_movie_session_by_id
+
+from db.models import Order, Ticket, User
+
+
+@transaction.atomic
+def create_order(tickets: list, username: str, date: str = None) -> None:
+    user = User.objects.get(username=username)
+    order_data = {"user": user}
+    if date:
+        order_data["created_at"] = date
+
+    order = Order.objects.create(**order_data)
+    order.save()
+
+    for ticket_data in tickets:
+        movie_session = get_movie_session_by_id(
+            ticket_data["movie_session"]
+        )
+        ticket_create = Ticket.objects.create(
+            movie_session=movie_session,
+            order=order,
+            row=ticket_data["row"],
+            seat=ticket_data["seat"]
+        )
+        ticket_create.save()
+
+
+def get_orders(username: str = None) -> QuerySet:
```

Comment:
Please indicate  QuerySet of which objects you expects here

Reply:
![image](https://github.com/mate-academy/py-tickets-and-orders/assets/134000226/d58b631b-06b9-488c-a609-54998532c0e8)